docs/traceflow-guide.md

```diff
@@ -178,3 +179,13 @@ therefore grant these ClusterRoles to any subject who may be responsible to
 troubleshoot the network. The admins may also decide to share the `view`
 ClusterRole to a wider range of subjects to allow them to read the traceflows
 that are active in the cluster.
+
+## Configuration Notes
+
+When traffic mode is `networkPolicyOnly`, it is recommended to specify the correct Pod CIDR (the aggregated cluster-wide
+Pod CIDR, not the per-Node Pod CIDRs) in the `podCIDR` field of `antrea-agent.conf`. This ensures that Traceflow functions
+correctly for inter-Node Pod-to-Pod traffic.For example:
+
+```text
+podCIDR: "10.10.0.0/16,fd00::/12"
```

Comment:
It would be better to have the ConfigMap context like other agent configs sample.

```suggestion
apiVersion: v1
kind: ConfigMap
metadata:
  name: antrea-config
  namespace: kube-system
data:
  antrea-agent.conf: |
    podCIDR: "10.10.0.0/16,fd00::/12"
```